### DISCUSSION TITLE
How to pass keepAlive properties in the JDBC Connection string

### DISCUSSION THREAD

**Discussion by oscartunjano** [jeremyevans/sequel#1984] [General]
Hello

I am using a JDBC connection string to connect to Oracle database. due to I have idle connections, I am trying to add parameters like "oracle.net.keepAlive" and "oracle.net.TCP_KEEPIDLE", but I noticed that the parameters has no effect. 

 I have found that keep alive by default will be configured using the operating system properties.
- net.ipv4.tcp_keepalive_time = 7200

I would like to change the parameter to 300 seconds, do you know how to override or pass this property from the JDBC connection string to send a keepalive packet every 300 seconds? 

additional information, I am using K8s due to that is not easy change the property from the operating system.

Best regards,

**Comment by jeremyevans** [comment]
Please show the code you are using.  Ideally, please also show how you would accomplish this using JDBC directly without using Sequel.  With that information, I should be able to help.
